locale/base-localization-file.yaml

```diff
@@ -0,0 +1,237 @@
+    redo:
+    Editor Themes:
+    Brew Editor:
+    Style Editor:
+    Properties:
+    jump to location in editor:
+    jump to location in preview:
+
+  brewTab:
+    snippets:
+      textEditor:
+      column break:
+      new page:
+      vertical spacing:
+      wide block:
+      image:
+      background image:
+      page number:
+      auto-incrementing page number:
+      link to page:
+      table of contents:
+      add comment:
+
+      phb:
+      spell:
+      spell list:
+      class feature:
+      note:
+      descriptive text box:
+      monster stat block:
+      wide monster stat block:
+      cover page:
+      artist credit:
+
+      tables:
+      class table:
+      half class table:
+      table:
+      wide table:
+      split table:
+
+  styleTab:
+    snippets:
+      styleEditor:
+      removeDropCap:
+      tweakDropCap:
+      addComment:
+
+      print:
+      a4PageSize:
+      squarePageSize:
+      inkFriendly:
+
+  propertiesTab:
+    brew:
+    title:
+    description:
+    thumbnail:
+    tags:
+    add tag:
+    systems:
+    5e:
+    4e:
+    3.5e:
+    pathfinder:
+    language:
+    languageSub: Sets the HTML Lang property for your brew. May affect hyphenation or spellcheck.
+    theme:
+    renderer:
+    Legacy:
+    v3:
+    legacyDemoLink: click here to see the demo page for the old legacy renderer!
+
+    authors: Authors
+    None.:
+    invited authors:
+    invite author:
+    invitedSub1: Invited authors are case sensitive.
+    invitedSub2: After adding an invited author, send them the edit link. there, they can choose to accept or decline the invitation.
+
+    privacy:
+    publish:
+    unpublish:
+    publishedSub: published homebrews will be publicly viewable and searchable (eventually...)
+    delete:
+    delete brew:
+    onlyAuthorDelete: Are you sure you want to delete this brew? Because you are the only owner of this brew, the document will be deleted permanently
+    confirm1: Are you REALLY sure? You will not be able to recover the document.
+    multipleAuthorDelete: Are you sure you want to remove this brew from your collection? This will remove you as an editor, but other owners will still be able to access the document.
+    confirm2: Are you REALLY sure? You will lose editor access to this document.
+    themesLegacy: Themes are not supported in the Legacy Renderer
+
+sharePage:
+  metadataWindow:
+    Authors:
+    no authors:
+    Systems:
+    no systems:
+    Tags:
+    no tags:
+    Description:
+    No description.:
+    Updated:
+
+  sourceDropdown:
+    source:
+    view:
+    download:
+    clone: clone to new
+
+accountPage:
+  Account Information:
+  username: "Username:"
+  noUser: No user currently logged in
+  lastLogin: "Last Login:"
+  Homebrewery Information:
+  brewsOnHomebrewery: "Brews on Homebrewery:"
+  Google Information:
+  linkedToGoogle: "Linked to Google:"
+  brewsOnGoogleDrive: "Brews on Google Drive:"
+  noFiles: Unable to retrieve files
```

Comment:
We should move colons to outisde the yaml, right?

Reply:
I'd say "yes", mostly because otherwise we need to remember to include them in every single localization.

Reply:
Fixed